element-android/pipeline.yml

```
@@ -39,6 +39,7 @@ steps:
     commands:
       - "./gradlew clean lintGplayRelease assembleGplayDebug --stacktrace"
       - "mv vector/build/outputs/apk/gplay/debug/*.apk ."
+      key: "elementApp"
```

I think this needs to be in the parent object:

```suggestion
    key: "elementApp"
```

the same for the other two places below.